django/db/backends/base/operations.py

```diff
@@ -686,8 +686,19 @@ def explain_query_prefix(self, format=None, **options):
             raise ValueError('Unknown options: %s' % ', '.join(sorted(options.keys())))
         return self.explain_prefix
 
-    def insert_statement(self, ignore_conflicts=False):
+    def insert_statement(self, on_conflicts=None):
         return 'INSERT INTO'
 
-    def ignore_conflicts_suffix_sql(self, ignore_conflicts=None):
+    def conflicts_suffix_sql(self, opts, fields, on_conflicts=None, update_fields=[]):
         return ''
+
+    def _get_unique_fields(self, opts, fields, update_fields):
+        unique_fields = []
+        for field in fields:
+            if field.unique and not field.primary_key and field.name not in update_fields:
+                unique_fields.append(field.name)
+
+        for unique_together in opts.unique_together:
+            unique_fields.extend(unique_together)
+
```

Comment:
You should also collect unique fields class based unique-constraints, you get can them from `opts.total_unique_constraints`.